### PR TITLE
Remove middle name from education verifier

### LINKED ISSUE #1229: Don't collect middle names for education verifier
Similar to #1196  – don't collect middle names for education verifier. SF-86 doesn't prompt for middle names for education verifier. eQIP can't collect them. 

screen shot 2018-11-28 at 12 40 51 pm

### PR DESCRIPTION
**Pull request by line47**
Fixes #1229 

screen shot 2018-12-03 at 11 14 57 am

### CODE REVIEW

**Comment by hursey013** [comment]
Looks like the education validator will need to have middle name removed in one of the test scenarios.

**Comment by hursey013** [comment]
I haven't looked yet, but do you know if this will require updates to the test json/xml?

**Comment by line47** [comment]
Ready for re-review 👍

File: api/testdata/complete-scenarios/test5.json [hunk 1]
```diff
@@ -6705,8 +6706,9 @@
                       "first": "",
                       "firstInitialOnly": false,
                       "middle": "",
+                      "hideMiddleName": true,
                       "middleInitialOnly": false,
-                      "noMiddleName": false,
+                      "noMiddleName": true,
```
Comment (hursey013) [inline api/testdata/complete-scenarios/test5.json]: To remain consistent with others:
```suggestion
                      "noMiddleName": false,
```

File: api/testdata/complete-scenarios/test5.json [hunk 2]
```diff
@@ -6853,8 +6855,9 @@
...
```
Comment (hursey013) [inline api/testdata/complete-scenarios/test5.json]: To remain consistent with others:
```suggestion
                      "noMiddleName": false,
```

**Comment by hursey013** [comment]
Seeing unrelated snyk errors now, going to rebase this branch and see if that does the trick.

**Pull request merged**